components/core/blocks/Microcart/Product.vue

```diff
@@ -71,22 +71,6 @@
               {{ (product.original_price_incl_tax ? product.original_price_incl_tax : product.price_incl_tax) * product.qty | price(storeView) }}
             </span>
           </div>
-          <div class="prices" v-else-if="isOnline && product.totals">
```

Comment:
Are you sure about that? `totals` has info about discount. How discount is displayed now? 🤔

Reply:
I agree with @gibkigonzo. We should not remove this block. Showing the discount per cart item is a good feature.
We are using it after applying a similar fix to this one (ie not adding tax on top of prices already including tax):
https://github.com/vuestorefront/vue-storefront/issues/4460#issuecomment-642709560